Xamarin.Forms.Platform.iOS/ContextScrollViewDelegate.cs

```diff
@@ -45,14 +45,15 @@ internal class ContextScrollViewDelegate : UIScrollViewDelegate
 		GlobalCloseContextGestureRecognizer _globalCloser;
 
 		bool _isDisposed;
-
+		UIScrollView _scrollView;
 		UITableView _table;
 
-		public ContextScrollViewDelegate(UIView container, List<UIButton> buttons, bool isOpen)
+		public ContextScrollViewDelegate(UIScrollView scrollView, UIView container, List<UIButton> buttons, bool isOpen)
```

Comment:
Passing scroll view so we can add closers to it and its cell on orientation change which causes dispose. Doing this also allows us to change the signature of several methods so that we no longer need to pass the scroll view over and over.